src/abstractdataframe/selection.jl

```diff
@@ -129,7 +129,7 @@ function select_transform!(nc::Pair{<:Union{Int, AbstractVector{Int}},
     if col_idx isa Int
         res = fun(df[!, col_idx])
     else
-        res = fun((cdf[i] for i in col_idx)...)
+        res = fun(i -> ntuple(cdf[col_idx[i]], length(col_idx))...)
```

Comment:
```suggestion
        res = fun(ntuple(i -> cdf[col_idx[i]], length(col_idx))...)
```

Reply:
Yes - I have changed this in parallel before I saw your post - it is fixed now. Thank you for catching it.